autotest/gcore/vsifile.py

```diff
@@ -758,6 +758,46 @@ def vsifile_19():
 
     return 'success'
 
+###############################################################################
+# Test gdal.VSIGetMemFileBuffer() and gdal.VSIFWriteL() reading buffers
+
+
+def vsifile_20():
+
+    import numpy.random as rand
```

Comment:
Test more complcated than needed. Instead of using a GDAL driver to create the file, can't you VSIFOpenL() it with "wb", write a few bytes in it and close ,